Applications/spire/source/charting/chart_view.cpp

```diff
@@ -5,7 +5,7 @@
 #include <QPainter>
 #include <QPaintEvent>
 #include "Nexus/Definitions/Money.hpp"
-#include "spire/charting/chart_model.hpp"
+#include "spire/charting/cached_chart_model.hpp"
```

Comment:
The ChartView shouldn't know or care about caching, remove any references to caching from the ChartView.